c/tskit/tables.c

```diff
@@ -50,7 +50,7 @@ typedef struct {
 
 typedef struct {
     const char *name;
-    void *array;
+    const void *array;
```

Comment:
This change is good, I'm just wondering: was it necessary to keep the compiler happy or did you spot the inconsistency and fix it?

Reply:
Compiler required it.  I am a bit concerned that the other arrays did not require similar casts, but decided to do the minimal necessary changes.